diff --git a/dist/esm/server/image-optimizer.js b/dist/esm/server/image-optimizer.js
index 4d686b24971db8f02c9ac5f8a2b86e844d377473..b1e53be6f38c49e315760fb7f142cdf9e19c7949 100644
--- a/dist/esm/server/image-optimizer.js
+++ b/dist/esm/server/image-optimizer.js
@@ -1138,7 +1138,6 @@ export async function fetchInternalImage(href, _req, _res, maximumResponseBody,
         const mocked = createRequestResponseMocks({
             url: href,
             method,
-            socket: _req.socket,
             maximumResponseBody
         });
         await handleRequest(mocked.req, mocked.res, parseReqUrl(href));
diff --git a/dist/server/image-optimizer.js b/dist/server/image-optimizer.js
index 7c827e0056adc336964fe1d6f8e45c8868dbf78f..d70ebd97ab3a461427b3911f631c2d309489cc31 100644
--- a/dist/server/image-optimizer.js
+++ b/dist/server/image-optimizer.js
@@ -1250,7 +1250,6 @@ async function fetchInternalImage(href, _req, _res, maximumResponseBody, handleR
         const mocked = (0, _mockrequest.createRequestResponseMocks)({
             url: href,
             method,
-            socket: _req.socket,
             maximumResponseBody
         });
         await handleRequest(mocked.req, mocked.res, (0, _url.parseReqUrl)(href));
